src/u_cx_at_params.c

```diff
@@ -51,7 +51,7 @@ static bool ipv4StringToIpAddress(const char *pAddressString,
     pIpAddress->address.ipv4 = 0;
 
     for (int i = 0; i < 4; i++) {
-        if (!isdigit(*pStrPtr)) {
+        if (!isdigit((unsigned char)*pStrPtr)) {
```

Comment:
But `isdigit()` is defined as `int isdigit( int ch );`:
https://en.cppreference.com/w/c/string/byte/isdigit
so there must be some weird config in the STM32 library.

Reply:
Ok, change to int then

Reply:
So it is happy with int but not `char`? How weird